app/models/alchemy/language.rb

```diff
@@ -19,38 +19,44 @@
 
 module Alchemy
   class Language < ActiveRecord::Base
-    belongs_to :site
-    has_many :pages
-
-    validates_presence_of :name
-    validates_presence_of :language_code
-    validates_presence_of :page_layout
-    validates_presence_of :frontpage_name
-    validates_uniqueness_of :language_code, scope: [:site_id, :country_code]
-    validate :presence_of_default_language
-    validate :publicity_of_default_language
+    validates :name, presence: true
+    validates :page_layout, presence: true
+    validates :frontpage_name, presence: true
+    validates :country_code, format: {
+        with: /\A[a-z]{2}\z/,
+        if: -> { country_code.present? }
+      }
```

Comment:
Style/IndentHash: Indent the right brace the same as the start of the line where the left brace is.